src/main/java/com/fasterxml/jackson/databind/cfg/MapperConfig.java

```diff
@@ -367,6 +367,18 @@ public BeanDescription introspectDirectClassAnnotations(Class<?> cls) {
 
     /**
      * Accessor for default property inclusion to use for serialization,
+     * considering possible per-type override for given base type and
+	 * possible per-type override for given property type.<br>
+     * NOTE: if no override found, defaults to value returned by
+     * {@link #getDefaultPropertyInclusion()}.
+     *
+     * @since 2.8.8
+     */
+    public abstract JsonInclude.Value getDefaultPropertyInclusion(Class<?> baseType,
```

Comment:
This is a breaking change, so could not introduce in a patch for sure, and not really in a patch either. Could of course add a new method, deprecate old one, add bridge method.
But what is the meaning of additional type argument here? Why is `baseType` not sufficient? I thought it was meant specifically as the type for lookups.